objects/AllPlayerCards.15bb07/ShortSupply.e5f541.ttslua

```diff
@@ -0,0 +1,36 @@
+local playmatAPI = require("playermat/PlaymatApi")
+
+function onLoad()
+  self.addContextMenuItem("Discard 10 cards", shortSupply)
+end
+
+-- called by context menu entry
+function shortSupply(color)
+  local matColor = playmatAPI.getMatColorbyPosition(self.getPosition())
+
+  -- get draw deck and discard position
+  local drawDeck = playmatAPI.getDrawDeck(matColor)
+  local discardPos = playmatAPI.getDiscardPosition(matColor)
+
+  -- error handling
+  if discardPos == nil then
+    broadcastToAll("Couldn't retrieve discard position from playermat!", "Red")
```

Comment:
Should probably limit the error to the calling player as well

Reply:
Will do